salt/basebox/config/lib-systemd-system-test.service

```diff
@@ -9,4 +9,4 @@ WantedBy={{ process }}-controller.target
 
 [Service]
 RestartSec=10
-ExecStart=/usr/bin/python3 /opt/{{ process }}.py $I
+ExecStart=/usr/bin/python3 /opt/{{ process }}.py %i
```

Comment:
... was wondering why I wasn't getting that blasted id coming though.

Reply:
also with the `onlyif` on the `disable`|`enable` commands, the configuration changes may not have been reloaded while testing so fiddling with the format wouldn't have had an effect until the number of processes was changed and triggered that state.